designing-reports/report-designer-tools/web-report-designer/how-to-implement-a-report-definition-storage.md

```diff
@@ -9,25 +9,28 @@ position: 5
 previous_url: /how-to-implement-a-report-definition-storage
 ---
 
-# Using Custom Report Definition Storage
```

Comment:
For SEO purposes, the Title and the # (Header) and Page_Title should differ. Now the Title and # are the same. Consider changing either of them, for example, by adding a word, etc.